examples/examples.js

```diff
@@ -31,7 +33,7 @@ const template = (
 );
 
 document.getElementById('iso-container').innerHTML = React.renderToString(template);
-React.render(template, document.getElementById('iso-container'));
+// React.render(template, document.getElementById('iso-container'));
```

Comment:
it is already, only for dev experience I wanted only one modal open at a time. This wasn't supposed to be committed.. reverting